src/components/shared/Error/Message.tsx

```diff
@@ -13,21 +14,24 @@ function Message({ error }: Props) {
 
     const intl = useIntl();
 
-    // Check if the message object is coming from the server
-    //  Easiest check I could think of. Our shortest keys are
-    //      `cta.foo` which can look a lot like how Supabase returns
-    //      errors when fetching wrong keys from a table (`tableName.col`)
-    //      so we can not use a RegEx like originally planned.
-    //  Code seems to almost always come back.
-    const messageFromServer =
-        typeof error === 'object' && error.hasOwnProperty('code');
+    const failedAfterRetry = retryAfterFailure(error?.message ?? error);
+
+    // Check if we have retried making the call or if we think the message object
+    //      is from Supabase
+    //  Initially was going to write a RegEx that detects if the mesaage
+    //      needs translated. However, Our shortest keys (cta.foo)
+    //      can also look a lot like how Supabase returns
+    //      errors when fetching wrong keys from a table (tableName.col).
+    const displayOnlyError =
```

Comment:
This is the smallest of nits, but I believe `displayErrorOnly` is a better phrasing of this variable name, aligning it with flags of a similar demarcation (e.g., `readOnly`, `nameOnly`, etc.).

Reply:
fixed